source/ProSymbolEditor/Models/SymbolAttributes/SymbolAttributeSet.cs

```diff
@@ -309,7 +442,139 @@ public void PopulateRowBufferWithAttributes(ref RowBuffer rowBuffer)
                 rowBuffer["countrycode"] = LabelAttributes.CountryCode;
             }
         }
-        
+
+        public void PopulateFeatureWithAttributes(ref Feature feature)
+        {
+            if (!string.IsNullOrEmpty(DisplayAttributes.Identity))
+            {
+                feature["identity"] = DisplayAttributes.Identity;
```

Comment:
Note: this was just a (low priority) observation - no need to change with this PR